CMSgov/design-system
# [WNMGDS-1179] - Standardize healthcare.gov icons

## Summary / Changed

- HhsLogo, WhiteHouseLogo, UsaLogo and MenuIcon added to core icon library
- Removed those items from subsystem and updated dependencies
- Added flag for use of error icon in InlineError component which utilizes the AlertCircleIcon component

## How to test

- Build and Start medicare to review the HhsIcon and the SimpleFooter in the docs
- Build and Start Healthcare to review the Footer content for logo updates, Header for menu icon updates, and all pages with errors to confirm they include the error icon.
- Build and Start core to confirm the addition of the menu icon to the icon docs
